### PR TITLE
Update hip-869.md

### PR DESCRIPTION
**Pull request by iwsimon**
Update HIP-869 with the latest protobufs.

**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-869.md [hunk 1]
```diff
@@ -101,141 +101,274 @@ Adopting a two-phase strategy, this approach facilitated the earlier release of
...
-service NodeService {
-		/**
-     * Prepare to add a new node to the network.
-     * When a valid council member initiates a HAPI transaction to add a new node,
-     * then the network should acknowledge the transaction and update the network’s Address Book within 24 hours.   
-     * The added node will not be active until the network is upgraded.
+service AddressBookService {
+    /**
+     * A transaction to create a new consensus node in the network.
+     * address book.
```
Comment (jsync-swirlds) [inline HIP/hip-869.md:107]:
```suggestion
     * A transaction to create a new consensus node in the network
     * address book.
```

File: HIP/hip-869.md [hunk 2]
```diff
@@ -101,141 +101,274 @@ Adopting a two-phase strategy, this approach facilitated the earlier release of
...
-     * Prepare to update the node to the network.
-	   * The node will not be updated until the network is upgraded.
+     * A transaction to update an existing consensus node from the network
+     * address book.
+     * <p>
+     * This transaction, once complete, SHALL modify the identified consensus
+     * node state as requested.
+     * <p>
+     * This transaction MAY be authorized by either the node operator OR the
+     * Hedera governing council.
```
Comment (jsync-swirlds) [inline HIP/hip-869.md:137]: This changed after we first wrote the text, it's only node operator (`admin_key`) now.
```suggestion
     * This transaction MUST be authorized by the node operator.
```

**Review by jsync-swirlds — APPROVED** [review]
Two small changes.
If we're trying to match what is _currently_ in the protobufs repo, these two suggestions may be ignored and I'll add them to my global update to be merged later (under a future "documentation update" HIP).

File: HIP/hip-869.md (hunk 1 above)
Comment (iwsimon) [reply HIP/hip-869.md]: Updated.

File: HIP/hip-869.md (hunk 2 above)
Comment (iwsimon) [reply HIP/hip-869.md]: Updated.

**Comment by iwsimon** [comment]
> Two small changes. If we're trying to match what is _currently_ in the protobufs repo, these two suggestions may be ignored and I'll add them to my global update to be merged later (under a future "documentation update" HIP).

I updated them here. Please update hedera-protobufs later. Thanks!

**Review by jsync-swirlds — APPROVED** [review]

**Review by mgarbs — APPROVED** [review]

**Pull request merged**